src/main/java/strategy/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package strategy;
+
+public interface OperationHandler {
+    int getAmount(int quantity);
+}
```

Comment:
Handler should process transaction by changing storage state accordingly
```suggestion
public interface OperationHandler {
    void processTransaction(FruitTransaction transaction);
}
```